antifeatures.toml

```diff
@@ -79,11 +79,11 @@ description.it = "Questo software non è più mantenuto. Ci si può aspettare ch
 icon = "user-times"
 title.en = "Package not maintained"
 title.eu = "Mantendu gabeko paketea"
-title.fr = "Paquet non maintenu"
+title.fr = "Package non maintenu"
 title.it = "Pacchetto non mantenuto"
-description.en = "This YunoHost package is not maintained and needs adoption."
+description.en = "This YunoHost package is not actively maintained and needs adoption. This means that minimal maintenance is made by volunteers who don't use the app, so you should expect the app to lose reliability over time. You can [learn how to package](https://yunohost.org/packaging_apps_intro) if you'd like to adopt it."
 description.eu = "Pakete honek ez du mantenduko duenik, boluntario baten beharra dauka."
-description.fr = "Ce package YunoHost n'est plus maintenu et doit être adopté."
+description.fr = "Ce package YunoHost n'est pas activement maintenu et a besoin d'être adopté. Cela veut dire que la maintenance minimale est réalisée par des bénévoles qui n'utilisent pas l'application, il faut donc s'attendre à ce que l'app perde en fiabilité avec le temps. Vous pouvez [apprendre comment packager](https://yunohost.org/packaging_apps_intro) si vous voulez l'adopter."
 description.it = "Questo pacchetto di YunoHost non è più mantenuto e necessita di essere adottato."
```

Comment:
(watdo about the .it and .eu string tho ? Imho I would remove those since they are completely out of sync)

Reply:
we could poke xabi (for eu) and tomi (for it) when we're decided on the phrasing?

Reply:
also @Psycojoker want to put those files on weblate

Reply:
@xabirequejo @xplosionmind do you want to update the basque and italian translations before merging?

Reply:
I won’t have access to a reliable computer until 18:00.